claimer/src/check-outstading-debts.ts

```diff
@@ -0,0 +1,40 @@
+import { gql } from 'graphql-request'
+import { logger } from './logger'
+
+const GET_OUTSTANDING_DEBTS = gql`
+  query {
+    relayPools {
+      items {
+        chainId
+        contractAddress
+        outstandingDebt
+        origins(where: { maxDebt_gt: "0" }) {
```

Comment:
ok so we're only using the "active" oirigins, which means that this will add a warning in datadog... until the current zora and lisk transactions are resolved.